src/sass/core/transform.scss

```diff
@@ -63,7 +55,17 @@ $-origin-opts: (
 			}
 		}
 
-		@each $rv in $-scale-range {
+		@each $rv in get.decimal-range(0, 1) {
+			// $rv : 0, 0.1, 0.2, 0.25, 0.3, 0.4, 0.5, 0.6, 0.7, 0.75, 0.8, 0.9, 1,
+			.scale-#{get.abs($rv * 100)} {
+				@include mixins.variants($composed...) {
+					--transform-scale: #{$rv};
+				}
+			}
+		}
+
+		@each $rv in get.decimal-range(1, 1) {
+			// $rv : 1.1, 1.2, 1.25, 1.3, 1.4, 1.5, 1.6, 1.7, 1.75, 1.8, 1.9, 2,
 			.scale-#{get.abs($rv * 100)} {
 				@include mixins.variants($composed...) {
 					--transform-scale: #{$rv};
```

Comment:
I want to combine these two `@each`. It's separated because the second `decimal-range` values are added `1` each from the origin range.
Can I do like this?
```
@for $i from 0 through 1 {
@each $rv in get.decimal-range(0, $i) {
			.scale-#{get.abs($rv * 100)} {
				@include mixins.variants($composed...) {
					--transform-scale: #{$rv};
				}
			}
}
```

Or is there better way to append those `$rv`s?